credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -133,17 +132,19 @@ def _pre_process(self, raw_data):
 
         >>> raw_data = '"password": "#####", "!@AAA12")'
         >>> print(self._pre_process(raw_data))
-        ['password', 'AAA12']
+        ['password','"#####', '!@AAA12']
         """
-        # r"((?<=').*?(?=')|(?<=\").*?(?=\")|[a-zA-Z0-9\._@-]+)"
-        # Match words and strings
-        # In strings, match the content starting from the first character
-        # e.g., "****" -> None
-        #       "$AAA123!!" -> AAA123!!
-        pattern = re.compile(
-            r"((?<=')\w\d.*?(?=')|(?<=\")\w\d.*?(?=\")|[\w\d]+)")
-        words = re.findall(pattern, raw_data)
-        return list(map(string_utils.snake_case_to_camel, words))
+        words = re.findall("(?<=').*?(?=')|(?<=\").*?(?=\")|[\w\d]+", raw_data)
+        strings = re.findall(r"(?<=').*?(?=')|(?<=\").*?(?=\")", raw_data)
```

Comment:
use `'` to define strings, and `"` for the docstrings (I know these strings were already defined with `"`, but please change it because it's not consistent with the rest of the codebase)

Reply:
what's the difference between strings and words? What should they contain?

Reply:
> what's the difference between strings and words? What should they contain?

snippet = `pwd = "123"` 
words = [pwd, 123]
strings = [123]

We will only convert that words that are not contained between " or ' into camel_case.